telescope/src/main/java/com/mattprecious/telescope/TelescopeLayout.java

```diff
@@ -444,11 +529,112 @@ private SaveScreenshotTask(Bitmap screenshot) {
 
     @Override protected void onPostExecute(File screenshot) {
       saving = false;
-      doneAnimator.start();
 
       if (lens != null) {
         lens.onCapture(screenshot);
       }
     }
   }
+
+  private void registerRequestCaptureReceiver() {
+    getContext().registerReceiver(requestCaptureReceiver, requestCaptureFilter);
+  }
+
+  private void unregisterRequestCaptureReceiver() {
+    getContext().unregisterReceiver(requestCaptureReceiver);
+  }
+
+  @TargetApi(LOLLIPOP) private void captureNativeScreenshot(final MediaProjection projection) {
+    capturingStart();
+
+    // Wait for the next frame to be sure our progress bars are hidden.
+    post(new Runnable() {
```

Comment:
Had a test screenshot today with the progress bars included. Posted all the work to the next frame to make sure the bars are hidden.